vector/src/main/java/im/vector/services/EventStreamService.java

```diff
@@ -873,15 +874,15 @@ public void refreshStatusNotification() {
             Log.d(LOG_TAG, "## refreshStatusNotification : put the service in foreground because of an initial sync " + mNotificationState);
 
             if (mNotificationState != NotificationState.INITIAL_SYNCING) {
-                startForeground(NOTIFICATION_ID, buildForegroundServiceNotification(getString(R.string.notification_sync_in_progress)));
+                startForeground(NOTIFICATION_ID_PERSISTENT, buildForegroundServiceNotification(getString(R.string.notification_sync_in_progress)));
```

Comment:
This notification should use the event notification id.
It replaces it during the application launch (see my previous comment).

Reply:
To fix this, I made the events notification come in over the PERSISTENT channel instead. Let me know if it still seems wrong.